### PR TITLE
Order summary

### PR DESCRIPTION
**Pull request by alisha331**

### CODE REVIEW

File: src/store/modules/cart.js [hunk 1]
```diff
@@ -27,11 +27,18 @@ const getters = {
     count: state => {
         return state.items.length
     },
+    productPrice: () => {
+        var price = 0;
+        state.products.forEach((obj) => {
```
Comment (rahimoddin) [inline src/store/modules/cart.js]: state.product ??
wont work please check again

File: src/store/modules/cart.js [hunk 2]
```diff
@@ -27,11 +27,18 @@ const getters = {
...
+        });
+        return price;
+    },
     totalPrice: () => {
         //TODO: add logic for total price
-        let price = 0;
-        state.items.map((item) => {
-            price = price + item.price * item.quantity;
+        var price = 0;
+        state.products.forEach((obj) => {
```
Comment (rahimoddin) [inline src/store/modules/cart.js]: state.product is not defined anywhere please revisit.

File: src/store/modules/cart.js (hunk 1 above)
Comment (alisha331) [reply src/store/modules/cart.js]: Incorporated

File: src/store/modules/cart.js (hunk 2 above)
Comment (alisha331) [reply src/store/modules/cart.js]: Incorporated

**Pull request merged**